src/Workspaces/CSharp/Portable/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -370,6 +379,19 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                 }
             }
 
+            // * { - in the switch expression context
+            if (currentToken.Kind() == SyntaxKind.OpenBraceToken && currentToken.Parent != null && currentToken.Parent.IsKind(SyntaxKind.SwitchExpression))
```

Comment:
```suggestion
            if (currentToken.IsKind(SyntaxKind.OpenBraceToken) && currentToken.Parent.IsKind(SyntaxKind.SwitchExpression))
```

Reply:
Is see both approaches being used throughout the code base, and it trips me up every time. I don't see a difference between usages, except for personal preference, is there?

My preference also goes to `.IsKind()`. I'll update the PR with that.

Reply:
`IsKind` is a bit cleaner, especially when compound conditions are involved. If someone sent a pull request that changed the use of `Kind` to `IsKind` for consistency, we'd likely merge it (as long as it only contained those changes and wasn't part of some other change).